.github/workflows/build.yml

```diff
@@ -217,7 +217,7 @@ jobs:
           # Decrypt the certificate
           openssl enc -aes-256-cbc -d -md sha512 \
             -k "${MACOS_CODESIGN_OPENSSL_PASSWORD}" \
-            -in /Users/runner/work/mixxx/mixxx/packaging/certificates/macos_developer_id_codesign_certificate.p12.enc \
+            -in "${{ github.workspace }}/packaging/certificates/macos_developer_id_codesign_certificate.p12.enc" \
             -out ~/certificate.p12
```

Comment:
When we need to replace the certificate anyway, I suggest to get rid of this. There is no benefit: The P12 certificate is already encrypted and protected with a passphrase. We should just base64-encode the certificate and store it in a Github secret instead of in the repository. Then we just do something like:

```bash
echo -n "$MACOS_CODESIGN_CERTIFICATE_BASE64" | base64 -d -o macos_codesign_certificate.p12
```

Removing the double encryption is not only much simpler, it also has another major benefit: By storing the certificate directly in the GitHub secrets instead of the repository, we can easily replace the certificate and rebuild old tagged versions later on by just triggering the GitHub action. Currently, this is not possible because we cannot add code to an existing tag.

Reply:
That's pretty much what I suggested in the Zulip channel :D

Reply:
OK, so let's remove the cert file from the git tree then.